# Generated by Django 2.2 on 2022-10-05 03:50

from django.db import migrations, models
import podomarket.validators


class Migration(migrations.Migration):

    dependencies = [
        ('podomarket', '0002_auto_20221005_0325'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='address',
            field=models.CharField(max_length=40, null=True, validators=[podomarket.validators.validate_no_special_characters]),
        ),
        migrations.AlterField(
            model_name='user',
            name='kakao_id',
            field=models.CharField(max_length=20, null=True, validators=[podomarket.validators.validate_no_special_characters]),
        ),
        migrations.AlterField(
            model_name='user',
            name='nickname',
            field=models.CharField(error_messages={'unique': '이미 사용중인 닉네임 입니다.'}, max_length=15, null=True, unique=True, validators=[podomarket.validators.validate_no_special_characters]),
        ),
    ]
